cms/static/js/spec/views/login_studio_spec.js

```diff
@@ -0,0 +1,38 @@
+define(['jquery', 'js/factories/login', 'common/js/spec_helpers/ajax_helpers', 'common/js/components/utils/view_utils'],
+function($, LoginFactory, AjaxHelpers, ViewUtils) {
+    'use strict';
+    describe("Studio Login Page", function() {
+        var submit_button;
+
+        beforeEach(function() {
+            loadFixtures('login.underscore');
+            /*jshint unused: false*/
+            var login_factory = new LoginFactory("/home/");
+            submit_button = $('#submit');
+        });
+
+        it('It will disable the submit button once it is clicked', function() {
+            spyOn(ViewUtils, 'redirect').andCallFake(function(){});
+            var addClassSpy = spyOn($.fn, 'addClass').andReturn(submit_button);
+            spyOn($.fn, 'removeClass').andReturn(submit_button);
+            var requests = AjaxHelpers.requests(this);
+            submit_button.click();
+            expect($.fn.addClass).toHaveBeenCalledWith('is-disabled');
```

Comment:
Why do you need a spy for checking to see if addClass was called? I would prefer:

    expect(...).toHaveClass('is-disabled')

IMO we should use spies sparingly and should instead test the desired behavior. What if the class was added through some other mechanism than ``addClass``, e.g. the HTML was re-rendered?